Doc for Quable (#3085)
ezsystems/developer-documentation: docs/product_catalog/quable/quable_guide.md

## Faster campaign execution

Product data defined in [[= pim_product_name =]] can be immediately used in [[= product_name =]] for building content and campaigns.
Marketing teams can create pages and enrich content using up-to-date product information, without the need to duplicate or manually synchronize data.

## Known limitations

The integration with [[= pim_product_name =]] has the following known limitations:

- It's not compatible with [Commerce](commerce.md) functionalities. [Carts](cart.md), [order management](order_management.md), and [shopping lists](shopping_list.md) can't be used with products coming from [[= pim_product_name =]].
- [Catalogs](product_catalog_guide.md#catalogs) can't be created from [[= pim_product_name =]] products.
- Product assets are not fully synchronized. Only the main asset thumbnail URL from [[= pim_product_name =]] is used.
- [Product-level access restrictions](policies.md#products) based on product type are not supported.

> - Products with unsupported identifiers (containing `/` or exceeding 64 characters) are not synchronized.

Reviewer: That's not exactly true.

If product contains `/` or it's identifier is longer than 64 characters, then certain "attachments" (availability, price management) won't work for it due to our arbitrary limitations (which might / are likely to be changed in upcoming versions).

```suggestion
```

Author: Changed to:
> - You can't define prices and availability for products with product codes exceeding 64 characters.
As discussed on Slack

---
description: The Quable product guide describes how you can use the product data from Quable in Ibexa DXP to create marketing campaigns built around your products.
month_change: true
---

# Quable product guide

## Overview

> [[= pim_product_name =]] integration connects [[= product_name =]] with [[[= pim_product_name =]] Product Information Management (PIM)](https://www.quable.com/en), making [[= pim_product_name =]] the authoritative source of product information for every website powered by [[= product_name =]].

Reviewer 2: ```suggestion
[[= pim_product_name =]] integration [add-on](<link to add on definition>) connects [[= product_name =]] with [[[= pim_product_name =]] Product Information Management (PIM)](https://www.quable.com/en), making [[= pim_product_name =]] the authoritative source of product information for every website powered by [[= product_name =]].
```

Author: I'll add this in a follow-up (together with the Add-ons table)

[[= pim_product_name =]] serves as the single source of truth for all product data, including attributes, classifications, variants, and translations.
[[= product_name =]] consumes this data and makes it available for use in content and digital experiences.

This approach eliminates the need to manage product data in multiple systems, while preserving a clear separation of responsibilities between product management and content usage.

## Availability

> Integration with [[= pim_product_name =]] PIM is available in all [[= product_name =]] editions, starting with [[= product_name =]] v5.0.7.

Reviewer 2: ```suggestion
The integration with [[= pim_product_name =]] PIM is available as an add-on for all [[= product_name =]] editions, starting with [[= product_name =]] v5.0.7.
```

Not "in", because you need to install and configure it.

> Before installing and enabling the integration, ensure that you have an active [[= pim_product_name =]] PIM instance with defined products, classifications, and channels.

Reviewer 2: ```suggestion
Before installing and enabling the add-on, ensure that you have an active [[= pim_product_name =]] PIM instance with defined products, classifications, and channels.
```

Then, [perform the initial configuration](install_quable.md).

## How does [[= pim_product_name =]] integration work

The integration is built on [[= product_name =]]'s [Remote PIM framework](add_remote_pim_support.md), which enables connection to external product data sources.

Once configured, the system performs:

- an initial synchronization of product data from Quable
- ongoing updates via webhooks (near real-time)

> Product data is mapped to the Ibexa DXP product data model, including variants, attributes and [product categorties](product_catalog_guide.md#product-taxonomy).

Reviewer 2: ```suggestion
Product data is mapped to the [[= product_name =]]'s product data model, including variants, attributes and [product categorties](product_catalog_guide.md#product-taxonomy).
```

This data is then available in the back office, content editing tools like [Online Editor](online_editor_guide.md) and [Page Builder](page_builder_guide.md), and APIs.

All product management operations remain handled in [[= pim_product_name =]].

## Capabilities

### Single source of truth

[[= pim_product_name =]] is the authoritative system for product data, including attributes, classifications, variants, and translations.

> All product management operations are performed in [[= pim_product_name =]].

Reviewer 2: You've just said the exact same words four lines above.
```suggestion
```

[[= product_name =]] consumes this data and makes it available for use within content and back office interfaces, enabling editorial teams to enrich content by reusing product information.

> ### Pricing and availability

Reviewer 2: This is not Quable's functionality, so I'd add it as a note, not as Quable intgration's capability.

Author: Moved to the section above, thanks

## Use cases

### Multi-market operations

A retailer operating across multiple markets can manage product data in [[= pim_product_name =]] using channels and localized languages.
[[= product_name =]] connects to the relevant channel and makes localized product information available for use in content and back office interfaces, ensuring consistency across markets from a single [[= pim_product_name =]] instance.

## Faster campaign execution

Product data defined in [[= pim_product_name =]] can be immediately used in [[= product_name =]] for building content and campaigns.
Marketing teams can create pages and enrich content using up-to-date product information, without the need to duplicate or manually synchronize data.

## Known limitations

The integration with [[= pim_product_name =]] has the following known limitations:

- It's not compatible with [Commerce](commerce.md) functionalities. [Carts](cart.md), [order management](order_management.md), and [shopping lists](shopping_list.md) can't be used with products coming from [[= pim_product_name =]].

> - [Catalogs](product_catalog_guide.md#catalogs) can't be created using Quable products.

Reviewer 2: ```suggestion
- [Catalogs](product_catalog_guide.md#catalogs) can't be created from [[= pim_product_name =]] products.
```

> - Product assets are not fully synchronized. Only the main asset thumbnail URL from [[= pim_product_name =]] is used.

Reviewer 2: ```suggestion
Only the main asset thumbnail URL from [[= pim_product_name =]] is used.
```

"main asset thumbnail URL" is hard to digest, but it could be me

Author: Reworked this into:
`- [Product assets](product_catalog_guide.md#product-assets) are not fully synchronized. Only the main product thumbnail from [[= pim_product_name =]] is used.`, hopefully it will be clearer

---
description: The Quable product guide describes how you can use the product data from Quable in Ibexa DXP to create marketing campaigns built around your products.

> month_change: false

Reviewer 3: ```suggestion
description: The Quable product guide describes how you can use the product data from Quable in Ibexa DXP to create marketing campaigns built around your products.
month_change: true
```

Reviewer 3: There is probably more missing `month_change: true`than those abvious ones.

Editorial teams that need to add DXP-specific content, such as promotional text or page layout, can still do so by reusing the product data, for example by using the [Online Editor](online_editor_guide.md#product-marketing-campaigns) and workflows including [collaborative editing](collaborative_editing_guide.md).

### Products, variants, and categories

[[= pim_product_name =]] product types map to [[= product_name =]] product types, and [[= pim_product_name =]] product variants are exposed as [[= product_name =]] product variants.

[[= pim_product_name =]]'s [product classifications](https://docs.quable.com/v5-EN/docs/documents-classification-new-version) are synchronized into [[= product_name =]]'s taxonomy, using product categories to organize products into structures.

### Prices, availability, and market context

You can use [[= product_name =]] to manage product availability and pricing for [[= pim_product_name =]]'s products, including creating advanced pricing strategies with [discounts](discounts.md) combined with [regions](product_catalog_guide.md#regions) and [currencies](product_catalog_guide.md#currencies).

> ## Known limitations

Reviewer 3: I feel like this isn't the place for this section as it get a bit too technical and detailed for a guide. But I see know better place.
As limitation to the capabilities, I would "hide" it a bit under this section.
```suggestion
### Known limitations
```
Or even move it down after Use cases?

Author: Moved down after use cases 👍

The integration with [[= pim_product_name =]] has the following known limitations:

- Product assets are not synchronized. Only the main asset thumbnail URL from [[= pim_product_name =]] is used as the product image. Asset collections, asset management, and variant thumbnails are not available.
- It's not possible to create [catalogs](product_catalog_guide.md#catalogs) with products coming from [[= pim_product_name =]].
- It's not possible to use the Commerce functionalities like [cart](cart.md), [order management](order_management.md), or [shopping lists](shopping_list.md) with products coming from [[= pim_product_name =]].

> - It's not possible to limit access to products using the existing [`ProductType` product limitations](policies.md#products).

Reviewer 3: The lake of counter proposal for those two reinforce my feeling that this shouldn't be there. IWe need something reassuring. If kept there, it's about what is covering the missing functionalities now. Are they covered by something else? Quable not compatible with Commerce because you can't have a cart to checkout would be weird. 

```suggestion
- It's not possible to use the [[= product_name_com =]] functionalities like [cart](cart.md), [order management](order_management.md), or [shopping lists](shopping_list.md) with products coming from [[= pim_product_name =]]. TODO: The cart is managed by Quable(?) Order by TODO…
- It's not possible to limit access to products using the existing [`ProductType` product limitations](policies.md#products).
```

Author: Reworded, as we've talked on Slack
